cfme/storage/volume.py

```diff
@@ -147,9 +147,10 @@ class VolumeAddEntities(View):
 
 
 class VolumeAddForm(View):
+    storage_manager = Select(name='storage_manager_id')
```

Comment:
This widget is only present in CFME 58z+.

Reply:
use BootstrapSelect

Reply:
done